core/cache/charm.go

```diff
@@ -31,6 +31,11 @@ type Charm struct {
 	details CharmChange
 }
 
+// Report returns information that is used in the dependency engine report.
+func (c *Charm) Report() map[string]interface{} {
+	return map[string]interface{}{}
```

Comment:
Nothing to report?

Reply:
Nothing of worth, I wanted a map rather than a slice, I guess I could have returned nil?